metrics/web/lib/project_groups/presentation/widgets/project_group_card.dart

```diff
@@ -144,6 +147,7 @@ class _ProjectGroupCardState extends State<ProjectGroupCard> {
     if (projectGroupsNotifier.deleteProjectGroupDialogViewModel == null) return;
 
     await showDialog(
+      barrierColor: barrierColor,
```

Comment:
Maybe we can extract the `showDialog` function call to the separate method like `_showProjectGroupDialog` to avoid code duplication?